README.md

```diff
@@ -24,15 +24,14 @@ Review the [Getting Started](https://github.com/vippsas/vipps-developers/blob/ma
 
 ## PSP API
 
-* [Vipps PSP API guide](vipps-psp-api.md): Developer guide for Vipps PSP API.
-* [Swagger](https://vippsas.github.io/vipps-psp-api) or  [ReDoc](https://vippsas.github.io/vipps-developer-docs/api/psp): PSP API reference documentation
-* [Vipps PSP API Checklist](vipps-psp-api-checklist.md): For PSP integrations.
+* [Guide](vipps-psp-api.md): Developer guide for Vipps PSP API.
```

Comment:
```suggestion
* [API guide](vipps-psp-api.md): Developer guide for Vipps PSP API.
```

I like explicitly mentioning "API", since we have the "How it works", and some people may be confused by just "guide".